weis/glue_code/gc_RunTools.py

```diff
@@ -1,5 +1,5 @@
-import os
-import matplotlib.pyplot as plt
+import matplotlib
```

Comment:
I don't think this is even used.  Better to just delete the import?

Reply:
You're right - I'm still confused as to why this was causing problems in the first case